design/sketch.webidl

```diff
@@ -344,12 +344,12 @@ enum GPUCullMode {
 };
 
 dictionary GPURasterizationStateDescriptor {
-    GPUFrontFace frontFace;
-    GPUCullMode cullMode;
+    GPUFrontFace frontFace = "cw";
```

Comment:
I don't think this is a good default. In most projects we use "ccw".
In general, I'd prefer us to *not* be opinionated where the default's aren't clear.

Reply:
+1, I missed this one the first time.

Reply:
The group resolved today that this field is required.